app/views/company/vacancies/index.html.erb

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
remove this extra line

Reply:
all files have a extra line on the botton

Reply:
In the bottom yes. Not in the top.

Reply:
done....